creation/server/creation.lua

```diff
@@ -35,42 +35,82 @@ function printoutHeader(name)
+    return printout
+  else
+    local printout = printoutHeader(zone.name)
+    printout = printout .. "CircleZone:Create("
+    printout = printout .. "vector3(" .. tostring(round(zone.center.x, 2)) .. ", " .. tostring(round(zone.center.y, 2))  .. ", " .. tostring(round(zone.center.z, 2)) .."), "
+    printout = printout .. tostring(zone.radius) .. ", "
+    printout = printout .. "{\n  name = \"" .. zone.name .. "\",\n  useZ = " .. tostring(zone.useZ) .. ",\n  --debugPoly = true\n})\n\n"
+    return printout
+  end
 end
 
 function parseBox(zone)
-  local printout = printoutHeader(zone.name)
-  printout = printout .. "BoxZone:Create("
-  printout = printout .. "vector3(" .. tostring(round(zone.center.x, 2)) .. ", " .. tostring(round(zone.center.y, 2))  .. ", " .. tostring(round(zone.center.z, 2)) .."), "
-  printout = printout .. tostring(zone.length) .. ", "
-  printout = printout .. tostring(zone.width) .. ", "
-  
-  printout = printout .. "{\n  name=\"" .. zone.name .. "\",\n  heading=" .. zone.heading .. ",\n  --debugPoly=true"
-  if zone.minZ then
-    printout = printout .. ",\n  minZ=" .. tostring(round(zone.minZ, 2))
-  end
-  if zone.maxZ then
-    printout = printout .. ",\n  maxZ=" .. tostring(round(zone.maxZ, 2))
+  if Config.ConfigFormat then
+    local printout = printoutHeader(zone.name)
+    printout = printout .. "coords = "
+    printout = printout .. "vector3(" .. tostring(round(zone.center.x, 2)) .. ", " .. tostring(round(zone.center.y, 2))  .. ", " .. tostring(round(zone.center.z, 2)) .."),\n"
+    printout = printout .. "length = " .. tostring(zone.length) .. ",\n"
+    printout = printout .. "width = " .. tostring(zone.width) .. ",\n"
+    printout = printout .. "name = \"" .. zone.name .. "\",\nheading = " .. zone.heading .. ",\n--debugPoly = true"
+    if zone.minZ then
+      printout = printout .. ",\nminZ = " .. tostring(round(zone.minZ, 2))
+    end
+    if zone.maxZ then
+      printout = printout .. ",\nmaxZ = " .. tostring(round(zone.maxZ, 2))
+    end
+    printout = printout .. "\n\n"
+    return printout
+  else
+    local printout = printoutHeader(zone.name)
+    printout = printout .. "BoxZone:Create("
+    printout = printout .. "vector3(" .. tostring(round(zone.center.x, 2)) .. ", " .. tostring(round(zone.center.y, 2))  .. ", " .. tostring(round(zone.center.z, 2)) .."), "
+    printout = printout .. tostring(zone.length) .. ", "
+    printout = printout .. tostring(zone.width) .. ", "
+    
+    printout = printout .. "{\n  name=\"" .. zone.name .. "\",\n  heading=" .. zone.heading .. ",\n  --debugPoly=true"
+    if zone.minZ then
+      printout = printout .. ",\n  minZ=" .. tostring(round(zone.minZ, 2))
+    end
+    if zone.maxZ then
+      printout = printout .. ",\n  maxZ=" .. tostring(round(zone.maxZ, 2))
+    end
+    printout = printout .. "\n})\n\n"
+    return printout
   end
-  printout = printout .. "\n})\n\n"
-  return printout
 end
```

Comment:
Since you're changing this file anyways, can you please add a trailing newline at the end here so git doesn't yell at us?